compiler/modules/front.syntax/tests/posTree.bal

```diff
@@ -0,0 +1,237 @@
+import wso2/nballerina.comm.diagnostic as d;
+
+type TerminalSyntaxNode TerminalSyntaxAstNode|FixedSyntaxNode|IdentifierSyntaxNode|StringLiteralSyntaxNode;
```

Comment:
These types all need to go in syntaxNode.bal

Reply:
fixed